Source/ProcessingTest/Presenters/ProcessingPresenter.cpp

```diff
@@ -178,7 +181,9 @@ void ProcessingPresenter::OpenRAWFile()
 {
     _view->EnableRendering(false);
 
-    QString fileName = QFileDialog::getOpenFileName(_view, tr("Open Image"), _lastDir, tr("DNG Files (*.dng *.DNG *.mlv *.MLV)"));
+    QString fileName = QFileDialog::getOpenFileName(_view, tr("Open Image"), _lastDir,
+    tr("DNG Files (*.dng *.DNG) Magiclantern files(*.mlv *.MLV)"));

```

Comment:
Seems like **;;** is missing between formats, can the filters be selected individually?

Reply:
updated